CHANGELOG.md

```diff
@@ -7,6 +7,61 @@ and is generated by [Changie](https://github.com/miniscruff/changie).
 
 
 
+## v0.12.0 - 2024-07-11
+
+### 🔥 Breaking Changes
+
+- api: Don't fallback to the default command on `stdout` and `stderr` by @helderco in https://github.com/dagger/dagger/pull/7857
+- api: Return absolute path on export instead of boolean by @helderco in https://github.com/dagger/dagger/pull/7500
+- api: Align `Container.withNewFile` signature with `Directory.withNewFile` by @helderco in https://github.com/dagger/dagger/pull/7293
+- api: Skip entrypoint by default in `withExec` by @helderco in https://github.com/dagger/dagger/pull/7136
+- sdks: Don't return Void by @helderco in @helderco in https://github.com/dagger/dagger/pull/7773
```

Comment:
```suggestion
- sdks: Don't return Void by @helderco in https://github.com/dagger/dagger/pull/7773
```

???

Reply:
Nope, seems totally reasonable to me :laughing: 

Will tidy this up, thanks @sagikazarmark :heart: